chatops4s-discord/src/main/scala/api/Server.scala

```diff
@@ -0,0 +1,167 @@
+package api
+
+import cats.effect.{ExitCode, IO, IOApp}
+import cats.implicits.toSemigroupKOps
+import io.circe.Json
+import models.{Button, DiscordResponse, InteractionContext, Message, MessageResponse}
+import org.http4s.HttpRoutes
+import org.http4s.blaze.server.BlazeServerBuilder
+import org.http4s.server.Router
+import sttp.tapir.*
+import sttp.tapir.json.circe.jsonBody
+import sttp.tapir.server.http4s.Http4sServerInterpreter
+import sttp.tapir.generic.auto.*
+import io.circe.generic.auto.*
+import org.bouncycastle.crypto.params.Ed25519PublicKeyParameters
+import org.bouncycastle.crypto.signers.Ed25519Signer
+import org.bouncycastle.util.encoders.Hex
+import sttp.tapir.EndpointOutput.OneOfVariant
+import sttp.model.StatusCode
+import io.circe.parser.*
+import sttp.tapir.swagger.bundle.SwaggerInterpreter
+import utilities.EnvLoader
+
+object Server extends IOApp:
```

Comment:
This one we have to split into three parts:
1. Endpoints definitions 
2. Service with the logic
3. Main that starts the server 

1 and 2 might be together for now (usually they get split if we care about openapi spec, but here we probably dont, as we implement predefined api)